Makefile

```diff
@@ -62,6 +63,6 @@ clean: clean-gemfile-lock
 
 ## Clean up the container specific Gemfile.lock
 clean-gemfile-lock:
-	rm -v ./docker/ruby-$(RUBY_VERSION)/Gemfile.lock
+	rm -v ./docker/ruby-$(RUBY_VERSION)/Gemfile.lock ||:
```

Comment:
never seen this combination before... makes sense to me

```
  - || means "if the left command fails, run the right command"
  - : is a shell builtin that does nothing and returns success (exit code 0)
```

ive always done `rm -v ./docker/ruby-$(RUBY_VERSION)/Gemfile.lock || 0` or something like that